lab-integration/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-integration",
+  "version": "1.0.0",
+  "scripts": {
+    "build": "npx tsc",
+    "start": "node dist/main.js",
+    "dev": "nodemon src/main.ts",
+    "lint": "eslint .  --ext .ts",
+    "lint-fix": "eslint --fix .  --ext .ts"
+  },
+  "devDependencies": {
+    "@stylistic/eslint-plugin-ts": "^1.7.0",
```

Comment:
It seems we need to pick between this plugin and prettier.

What do you think?

https://eslint.style/guide/why

Reply:
Once decided, we need to run a linter or a formatter to fix the rest of the files.